sqlalchemy_bigquery/__init__.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2017 The sqlalchemy-bigquery Authors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+from .version import __version__
+
+from .base import BigQueryDialect
+from .base import (
+    STRING,
+    BOOL,
+    BOOLEAN,
+    INT64,
+    INTEGER,
+    FLOAT64,
+    FLOAT,
+    TIMESTAMP,
+    DATETIME,
+    DATE,
+    BYTES,
+    TIME,
+    RECORD,
+    NUMERIC,
+    BIGNUMERIC,
+)
+
+__all__ = [
+    "__version__",
+    "BigQueryDialect",
+    "STRING",
+    "BOOL",
+    "BOOLEAN",
+    "INT64",
+    "INTEGER",
+    "FLOAT64",
+    "FLOAT",
+    "TIMESTAMP",
+    "DATETIME",
+    "DATE",
+    "BYTES",
+    "TIME",
+    "RECORD",
+    "NUMERIC",
+    "BIGNUMERIC",
+]
+
+try:
+    import pybigquery
+except ImportError:
+    pass
+else:  # pragma: NO COVER
+    if not hasattr(pybigquery, "__version__"):
```

Comment:
I'm curious what the `not hasattr` is doing. Could you add a comment about this? If `__version__` _is_ present it's less likely to conflict?

Reply:
That's a holdover from the shim, I think.  The shim would have that attribute.  I'll get rid of it.

Reply:
fixed